scripts/vsphere/README.md

```diff
@@ -0,0 +1,58 @@
+# Attach VM to Anthos Service Mesh
```

Comment:
Can you add (private preview) or something to indicate that it's 1) not GA and 2) won't work for the public?